resources/assets/components/FilterBar/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import DropdownFilter from '../DropdownFilter';
+import MultiValueFilter from '../MultiValueFilter';
+
+class FilterBar extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      status: 'accepted',
+      tags: {
+        'good-photo': false,
+        'good-quote': false,
+        'hide-in-gallery': false,
+        'good-for-sponsor': false,
+        'good-for-storytelling': false,
+      }
+    };
+
+    // The component provides a function the children components can use to update the state.
+    this.updateFilters = this.updateFilters.bind(this);
+  }
+
+  updateFilters(value){
+    if (['pending', 'accepted', 'rejected'].includes(value)) {
+      this.setState({
+        status: value,
+      });
+    }
+
+    if (['good-photo', 'good-quote', 'hide-in-gallery', 'good-for-sponsor', 'good-for-storytelling'].includes(Object.keys(value)[0])) {
+      let key = Object.keys(value)[0];
+
+      this.setState((previousState) => {
+        const newState = {...previousState};
+        newState.tags[key] = Object.values(value)[0];
+
+        return newState;
+      });
+    }
+  }
+
+  render() {
+    // ***Rendering the children**
+
+    // In this case DropdownFilter and MultiValueFilter are children of the FilterBar as layed out in Campaign Single.
```

Comment:
@sbsmith86 I kept your comments/explanation because I thought they were super helpful and could be good context for the future!

Reply:
Cool! maybe just keep lines 48 and 49? this one is just explaining how we are using this component in `CampaignSingle` but we want to keep this general because it could be used differently some place else.